td.vue/tests/unit/desktop/menu.spec.js

```diff
@@ -4,6 +4,7 @@ import deu from '@/i18n/de.js';
 import ell from '@/i18n/el.js';
 import eng from '@/i18n/en.js';
 import id from '@/i18n/id.js';
+import ms from '@/i18n/ms.js';
```

Comment:
ideally both id.js and ms.js would be imported in alphabetical order, so could these both be moved to after hi.js